Lib/pdb.py

```diff
@@ -1073,6 +1086,24 @@ def _select_frame(self, number):
         self.print_stack_entry(self.stack[self.curindex])
         self.lineno = None
 
+    def do_exceptions(self, arg):
+        if not arg or arg == "list":
```

Comment:
The `list` arg is not documented. Consider the fact that `break` command does not have a `list` option, maybe we can simply remove it.